### DISCUSSION TITLE
What is 8964?

### DISCUSSION THREAD

**Discussion by KexunNiu** [CMPUT301F21T24/Habot#32] [Q&A] [answered]
I don't know, can someone explain it?

**Comment by kkli08** [comment, marked as answer]
It is possible that you may be referring to the Tiananmen Square protests of 1989, which are also known as the June Fourth Incident (六四事件) or the 1989 Democracy Movement. The protests, which took place in Beijing's Tiananmen Square, were a student-led pro-democracy movement that called for political reform and an end to corruption in the Chinese government. The protests were eventually suppressed by the Chinese government, resulting in a violent crackdown and an unknown number of casualties. The exact number of casualties is not known, but estimates range from several hundred to several thousand. The Chinese government has since censored any mention of the event and has not allowed public discussion or commemoration of the incident.
